src/commands/seek.rs

```diff
@@ -31,17 +31,16 @@ pub async fn seek(
             .and_then(|token| token.parse::<u64>().ok()),
     );
 
-    if minutes.is_none() || seconds.is_none() {
-        return create_response(&ctx.http, interaction, FAIL_TIMESTAMP_PARSING).await;
-    }
+    verify(minutes, ParrotError::Other(FAIL_TIMESTAMP_PARSING))?;
+    verify(seconds, ParrotError::Other(FAIL_TIMESTAMP_PARSING))?;
```

Comment:
Can't we do this check in the same line?
```suggestion
    verify(minutes && seconds, ParrotError::Other(FAIL_TIMESTAMP_PARSING))?;
```

Reply:
No. That does not compile.

Reply:
They are both options, each can be converted to a bool, but not implicitly. What could be done is 

```rust
verify(minutes.to_bool() && seconds.to_bool(), ParrotError::Other(FAIL_TIMESTAMP_PARSING))?;
```

Reply:
But does that not fail when one of them is not defined? I would prefer that over two different checks for the same error.

Reply:
If anything, this is a little better no? I have no strong opinions on either.

```rust
verify(minutes.is_none() && seconds.is_none(), ParrotError::Other(FAIL_TIMESTAMP_PARSING))?;
```

Reply:
Well, yes that one is better, but then we need to negate the verification.

Reply:
Ah, true. Edited.

Reply:
Sure, just defeats the purpose of implementing ToBool trait for options. I might be able to cook something up where instead of receiving a ToBool it receives an expression where elements implement ToBool. I will try it later tonight

Reply:
But there are multiple occurrences where you need to validate more than one thing for the same error, which is why I asked!

Reply:
Took the opportunity to have more specific messages.